src/messages.rs

```diff
@@ -110,23 +110,51 @@ pub enum Message {
     /// This is emitted at the end of the build as the last message.
     /// Added in Rust 1.44.
     BuildFinished(BuildFinished),
+    /// A line of text which isn't a cargo or compiler message.
+    /// Line separator is not included
+    TextLine(String),
```

Comment:
Please apply `#[serde(skip)]` to this variant, we should never see a `text-line` from cargo

Reply:
Huh, just slapping `skip` broke serde: https://github.com/matklad/cargo_metadata/commit/0b1c8d85c860fe9fb3488fea5990d659780065fc

I think it intrracts badly with `#[serde(other)]` below. I'll try to read on `other` and figure out how to convince serde do what we want to do here.

Reply:
Ah, that was genuine serde bug: https://github.com/serde-rs/serde/issues/1804

Bumped serde-derive req in Cargo.toml